pkg/client/buildconfigs.go

```diff
@@ -37,10 +37,10 @@ func newBuildConfigs(c *Client, namespace string) *buildConfigs {
 }
 
 // List returns a list of buildconfigs that match the label and field selectors.
-func (c *buildConfigs) List(label, field labels.Selector) (result *buildapi.BuildConfigList, err error) {
+func (cfg *buildConfigs) List(label, field labels.Selector) (result *buildapi.BuildConfigList, err error) {
```

Comment:
Shorter type reference variables are usually better.  Why is cfg better than c?  This is more of a client interface and a cfg interface.